hasura/migrations/default/1703175768541_create_roles_table/up.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE "public"."roles" (
+  "id" varchar(50) NOT NULL DEFAULT gen_random_friendly_id ('rol'),
+  "created_at" timestamptz NOT NULL DEFAULT now(),
+  "updated_at" timestamptz NOT NULL DEFAULT now(),
+  "user_id" varchar(50) NOT NULL,
+  "role" text NOT NULL,
+  "team_id" varchar(50) NOT NULL,
+  PRIMARY KEY ("id"),
+  FOREIGN KEY ("user_id") REFERENCES "public"."user" ("id") ON UPDATE RESTRICT ON DELETE CASCADE,
+  FOREIGN KEY ("role") REFERENCES "public"."role" ("value") ON UPDATE RESTRICT ON DELETE CASCADE,
```

Comment:
```suggestion
  FOREIGN KEY ("role") REFERENCES "public"."role" ("value") ON UPDATE RESTRICT ON DELETE RESTRICT,
```
for this one in particular a safer approach seems best